client/src/hooks/graphql/use-with-record-state.tsx

```diff
@@ -342,7 +342,7 @@ export function useWithRecordState(
     upload(mentor._id, answer.answer.question, answer.recordedVideo, trim);
   }
 
-  function cancelUploadVideo(task: UploadTask) {
+  function cancelUploadVideo(task: UploadTask | undefined) {
```

Comment:
shouldn't this be 

```typescript
function cancelUploadVideo(task?: UploadTask) {
```

?

...curious also what would be the case where task param would be null?